processor/transformer/transformer.go

```diff
@@ -310,6 +315,39 @@ func (t *HTTPLBTransport) NewRoundTripper(scheme, target string, config httplb.T
 	return httplb.RoundTripperResult{RoundTripper: t.Transport, Close: t.CloseIdleConnections}
 }
 
+func (trans *handle) getPicker() func(prev picker.Picker, allConns conn.Conns) picker.Picker {
```

Comment:
we are adding method that contain logic specific to httplb, to a generic component.

It would be preferable to encapsulate this in a separate component. A simple wrapper on httplb.NewClient would be enough.

Something like

```go
HTTPLBFromConfig(c *config.Config) httplb.Client {
pickerType := config.GetString("Transformer.Client.httplb.pickerType", "power_of_two")
...

return httplb.NewClient(....
}
```

Reply:
Done... is this behaviour what you expected?